### PR TITLE
update-publication-date

### PR DESCRIPTION
**Pull request by corneelwille**

### CODE REVIEW

File: content/json-ld/latest/events/update-publication-date.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+---
+
+# Update the publication date
```
Comment (LucWollants) [inline content/json-ld/latest/events/update-publication-date.md:4]: Are we gonna name this publication date or available from?

File: content/json-ld/latest/events/update-publication-date.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+PUT /events/{eventId}/available-from
+```
+
+## Request headers
+
+| Header        | Value            | Required? |
+| ------------- | ---------------- | --------- |
+| Authorization | Bearer {token}   | true      |
+| X-Api-Key     | {apiKey}         | true      |
+| Content-Type  | application/ld+json;domain-model=Publish | true |
```
Comment (LucWollants) [inline content/json-ld/latest/events/update-publication-date.md:22]: Content type is just `applicaiton/json`

File: content/json-ld/latest/events/update-publication-date.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
+---
+---
+
+# Update the publication date
+
+Once set, it is possible to change the publication date of an event to a date equal to the now-date or a date in the future.
```
Comment (LucWollants) [inline content/json-ld/latest/events/update-publication-date.md:6]: I would:
- remove `Once set`
- change `now-date` to `current date`

File: content/json-ld/latest/events/update-publication-date.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+The following requests are example requests
+
+
+**date in the past**
+
+In the example below, a date in the past is used, so the new availableFrom will be automatically set to the now-date
+```
+PUT https://io-test.uitdatabank.be/events/03116768-1abc-405a-93d7-ba6ede52fe09/available-from
+Content-Type: application/ld+json;domain-model=Publish
```
Comment (LucWollants) [inline content/json-ld/latest/events/update-publication-date.md:56]: The type is just `application/json`

File: content/json-ld/latest/events/update-publication-date.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+## Example
+
+### request
+
+The following requests are example requests
+
+
+**date in the past**
+
+In the example below, a date in the past is used, so the new availableFrom will be automatically set to the now-date
```
Comment (LucWollants) [inline content/json-ld/latest/events/update-publication-date.md:53]: Replace `now-date` with ` current date`

File: content/json-ld/latest/events/update-publication-date.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+X-Api-Key: {apiKey}
+
+{
+  "availableFrom": "1950-10-01T12:00:00+00:00"
+}
+```
+
+**date in the future**
+
+In the example below, a date in the future is used, meaning that the event will not be published in the API until that date.
```
Comment (LucWollants) [inline content/json-ld/latest/events/update-publication-date.md:67]: The part `published in the API` sounds a bit weird, is just `published` enough?

**Comment by LucWollants** [comment]
@corneelwille the remarks given for the event endpoint also apply to the place endpoint.

File: content/json-ld/latest/events/update-publication-date.md (hunk 1 above)
Comment (corneelwille) [reply content/json-ld/latest/events/update-publication-date.md]: Changed it to available from for consistency 👍

File: content/json-ld/latest/places/update-available-from.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
+---
+---
+
+# Update the publication date
```
Comment (LucWollants) [inline content/json-ld/latest/places/update-available-from.md:4]: This still contains publication date

File: content/json-ld/latest/events/update-available-from.md [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+  "availableFrom": "1950-10-01T12:00:00+00:00"
+}
+```
+
+**date in the future**
+
+In the example below, a date in the future is used, meaning that the event will not be published until that date.
+```
+PUT https://io-test.uitdatabank.be/events/03116768-1abc-405a-93d7-ba6ede52fe09/available-from
+Content-Type: application/ld+json;domain-model=Publish
```
Comment (LucWollants) [inline content/json-ld/latest/events/update-available-from.md:70]: This still contains the wrong content type `application/ld+json;domain-model=Publish`

**Pull request merged**